lib/project_types/node/commands/deploy/heroku.rb

```diff
@@ -33,17 +33,15 @@ def call(_args, _name)
           spin_group.wait
 
           if (account = heroku_service.whoami)
-            spin_group.add("Authenticated with Heroku as `#{account}`") { true }
-            spin_group.wait
+            @ctx.puts("{{v}} Authenticated with Heroku as `#{account}`")
           else
             CLI::UI::Frame.open("Authenticating with Heroku…", success_text: '{{v}} Authenticated with Heroku') do
               heroku_service.authenticate
             end
           end
 
           if (app_name = heroku_service.app)
-            spin_group.add("Heroku app `#{app_name}` selected") { true }
-            spin_group.wait
+            @ctx.puts("{{v}} Heroku app `#{app_name}` selected")
```

Comment:
This is weird, I wonder why they were spin groups before

Reply:
Maybe because they were already being used, so we just kept using them? 🤷